src/bindings/python/PyBuiltinTransformRegistry.cpp

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright Contributors to the OpenColorIO Project.
+
+#include "PyOpenColorIO.h"
+#include "PyUtils.h"
+
+namespace OCIO_NAMESPACE
+{
+
+void bindPyBuiltinTransformRegistry(py::module & m)
+{
+    // Wrapper to preserve the BuiltinTransformRegistry singleton.
+    class Wrapper
+    {
+    public:
+        Wrapper() = default;
+        Wrapper(const Wrapper &) = delete;
+        Wrapper & operator=(const Wrapper &) = delete;
+        ~Wrapper() = default;
+
+        size_t getNumBuiltins() const noexcept
```

Comment:
We should make this interface more Pythonic. This could probably be boiled down a single tuple iterator. I can propose that in a later PR.

Reply:
Good point. In a Later PR.